frontend/src/components/companyList/CompanyCard.tsx

```diff
@@ -1,15 +1,26 @@
 import { motion } from 'framer-motion'
 import { useState } from 'react'
-import SpringModal from '../modal/SpringModal'
+import {SpringModal} from '@/components/modal/SpringModal'
 import type { CompanyProps } from '@/lib/interfaces'
 
-const CompanyCard = ({company, userId}: {company: CompanyProps, userId: string}) => {
+const CompanyCard = ({
```

Comment:
Rather than an additional line at the end of the file to export, you can export this in the same line you declare it

```suggestion
export const CompanyCard = ({
```